src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -122,12 +122,16 @@ public function load(array $configs, ContainerBuilder $container)
             foreach ($bundle['paths'] as $path) {
                 $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($path, $namespace));
             }
+
+            $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($config['default_path'].'/bundles/'.$namespace, $namespace));
```

Comment:
Following the actual convention `'/Resources/'.$name.'/views'` then should be `'/bundles/'.$name`?